home/templates/home/index.html

```diff
@@ -5,73 +5,84 @@
 
     <div class="row landing-page">
         <div class = "col-12 col-sm-12  col-md-12 col-lg-6 " id="accordion-wrapper">
-            <h1 class="text-center">Upcoming Hackathons</h1>
+            <h1 class="text-center mt-4">Upcoming Hackathons</h1>
             <div class="accordion" id="accordion-upcoming-hackathons">
                 {% if hackathons %}
-                    {% for hackathon in hackathons %}
-                        {% if hackathon.status == 'published' %}
-                            {% if hackathon.organisation == 'Code Institute' or hackathon.organisation == user.organisation %}
-                                {% if not hackathon.status == 'deleted' %}
-                                <div class="card">
-                                    <div class="card-header" id="heading-{{ hackathon.id }}">
-                                        <span class="btn btn-link" type="button" data-toggle="collapse" data-target="#collapse-{{ hackathon.id }}" aria-expanded="true" aria-controls="collapse-{{ hackathon.id }}">
-                                            <h5 class="card-title"><i class="fas fa-caret-down"></i> {{ hackathon.display_name }}</h5>
-                                        </span>
-                                    </div>
-                                    <div id="collapse-{{ hackathon.id }}" class="collapse" aria-labelledby="heading-{{ hackathon.id }}" data-parent="#accordion-upcoming-hackathons">
-                                        <div class="card-body">
-                                        
-                                            <h6 class="card-subtitle mb-2 text-muted"><span class="drk-txt-hkth">Start: </span>{{ hackathon.start_date }} - <span class="drk-txt-hkth">End: </span>{{ hackathon.end_date }}</h6>
-                                                <p class="card-text">{{ hackathon.description }}</p>
-                                            {% if hackathon.organiser %}
-                                            <p><span class="drk-txt-hkth">Organiser: </span><a href="#" class="p-blue card-link">{{ hackathon.organiser }}</a></p>
-                                            {% endif %}
-                                            <a href="#" class="btn btn-ci mr-3">Read More</a>
-                                        </div>
+                {% for hackathon in hackathons %}
+                    {% if hackathon.organisation == 'Code Institute' or hackathon.organisation == user.organisation %}

```

Comment:
Maybe we can change this to not actually hard code `Code Institute`, but use the first organisation `hackathon.organisation.id == 1`?

Reply:
Okay